Multisig/Logic/Models/Signer.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Signer.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 25.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Web3
+
+class Signer {
+    struct Signature: Equatable {
+        var value: String
+        var signer: String
+    }
+
+    /// Signs the hash of the provided string with a stored private key.
+    /// Currently the app can store only one private key.
+    /// - Parameters:
+    ///   - string: string to hash and sign
+    /// - Throws: errors during sisgning process
+    /// - Returns: Signature object containing hex(r) hex(s) hex(v + 27) as one strig of secp256k1 signature
+    static func sign(_ string: String) throws -> Signature {
+        let hash = EthHasher.hash(string.data(using: .ascii)!)
```

Comment:
I would use `.utf8` encoding instead of ascii encoding

Reply:
Ok, I can change. It should not make any change in logic.